lib/user.rb

```diff
@@ -0,0 +1,22 @@
+class User
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(user, joke)
+    user.jokes << joke
```

Comment:
This definitely works, but keep in mind that we have a `learn` method for this exact thing - so it could look like `user.learn(joke)`